submissions/xT0PERx/Popup-style/index.html

```diff
@@ -4,130 +4,134 @@
-                        <img src="images\apps.png" />
-                        <p>Apps</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\blogs.png" alt="" />
-                        <p>Blogs</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\calendar.png" />
-                        <p>Calendar</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\chat.png" alt="" />
-                        <p>Chat</p>
-                    </div>
-
-                    <div tabindex="0">
-                        <img src="images\classroom.png" />
-                        <p>Classroom</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\contacts.png" alt="" />
-                        <p>Contacts</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\documents.png" />
-                        <p>Documents</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\files.png" alt="" />
-                        <p>Files</p>
-                    </div>
-
-                    <div tabindex="0">
-                        <img src="images\mail.png" />
-                        <p>Mail</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\maps.png" alt="" />
-                        <p>Maps</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\mic.png" />
-                        <p>Microphone</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\news.png" alt="" />
-                        <p>News</p>
-                    </div>
-
-                    <input id="second" type="checkbox" />
-                    <label for="second">More</label>
+        <nav class="container">
+            <ol>
+                <li>
+                    <a class="mail" href="#" tabindex="1"><p>Mail</p></a>
+                </li>
+                <li>
+                    <a class="images" href="#" tabindex="2"><p>Images</p></a>
+                </li>
+                <li class="main_tullbar">
+                    <input
+                        tabindex="3"
+                        id="first"
+                        class="check_input"
+                        type="checkbox"
+                    />
+                    <label class="menu" for="first"> </label>
 
-                    <section class="icons2">
-                        <div tabindex="0">
-                            <img src="images\photos.png" />
-                            <p>Photos</p>
+                    <div class="icons">
+                        <div tabindex="4">
```

Comment:
All items inside the popup must be a link. In that case, you don't need to use `tabindex` attribute.

Reply:
This means that I need to add the <a> tag in the <div> tags of my pop-up window, and put <img> in it, and then the need to specify the tabindex property disappears? Did I understand correctly?

Reply:
Yes, put `img` and `p` into `a` tag and remove `tabindex` attribute.

Reply:
Is it just incorrect to set the index “0” for my “divs”?

Reply:
If you add link `a` tag you no need to use `tabindex` at all
For example see google realization: [Video](https://share.vidyard.com/watch/3XwULvWkezXGtbEoJGmfks?)